playground_test.go

```diff
@@ -106,18 +110,24 @@ type GetProjectResponse struct {
 }
 
 const MutationUpdateProjectPersist = `
-mutation($projectId: UUID!, $persist: Boolean!) {
-  updateProject(input: { id: $projectId, persist: $persist }) {
+mutation($projectId: UUID!, $title: String!, $description: String!, $readme: String!, $persist: Boolean!) {
+  updateProject(input: { id: $projectId, title: $title, description: $description, readme: $readme, persist: $persist }) {
     id
+		title
```

Comment:
There is a tabs vs. spaces issue here. I suggest converting to spaces, to match the lines above and below.